lux/core/series.py

```diff
@@ -55,3 +54,94 @@ def f(*args, **kwargs):
+                    return ""
+
+                if len(self) <= 0:
+                    warnings.warn(
+                        "\nLux can not operate on an empty series.\nPlease check your input again.\n",
+                        stacklevel=2,
+                    )
+                    print(series_repr)
+                    return ""
+                ldf.maintain_metadata()
+
+                if lux.config.default_display == "lux":
+                    self._toggle_pandas_display = False
+                else:
+                    self._toggle_pandas_display = True
+
+                # df_to_display.maintain_recs() # compute the recommendations (TODO: This can be rendered in another thread in the background to populate self._widget)
+                ldf.maintain_recs()
+
+                # Observers(callback_function, listen_to_this_variable)
+                ldf._widget.observe(ldf.remove_deleted_recs, names="deletedIndices")
+                ldf._widget.observe(ldf.set_intent_on_click, names="selectedIntentIndex")
+
+                if len(ldf.recommendation) > 0:
+                    # box = widgets.Box(layout=widgets.Layout(display='inline'))
+                    button = widgets.Button(
+                        description="Toggle Pandas/Lux",
+                        layout=widgets.Layout(width="140px", top="5px"),
+                    )
+                    ldf.output = widgets.Output()
+                    # box.children = [button,output]
+                    # output.children = [button]
+                    # display(box)
+                    display(button, ldf.output)
+
+                    def on_button_clicked(b):
+                        with ldf.output:
+                            if b:
+                                self._toggle_pandas_display = not self._toggle_pandas_display
+                            clear_output()
+                            if self._toggle_pandas_display:
+                                print(series_repr)
+                            else:
+                                # b.layout.display = "none"
+                                display(ldf._widget)
+                                # b.layout.display = "inline-block"
+
+                    button.on_click(on_button_clicked)
+                    on_button_clicked(None)
+                else:
+                    warnings.warn(
+                        "\nLux defaults to Pandas when there are no valid actions defined.",
+                        stacklevel=2,
+                    )
+                    print(series_repr)
+
+        except (KeyboardInterrupt, SystemExit):
+            raise
+        except:
+            warnings.warn(
+                "\nUnexpected error in rendering Lux widget and recommendations. "
+                "Falling back to Pandas display.\n\n"
+                "Please report this issue on Github: https://github.com/lux-org/lux/issues ",
+                stacklevel=2,
+            )
+            print(series_repr)
```

Comment:
Is this print statement necessary?

Reply:
Yes, it's equivalent to `display` in `_repr_html_`